aiormq/channel.py

```diff
@@ -196,21 +197,23 @@ async def rpc(
                 )
 
                 self.__close_event.set()
-                self.write_queue.put_nowait(
-                    ChannelFrame.marshall(
```

Comment:
I suddenly realized that awaiting for a regular put methot would be even better

Reply:
Ok, changed it.